app/controllers/api/v1/ami_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Api::V1
+  # API to query for Area Median Income values
+  class AmiController < ApiController
+    def get
```

Comment:
@akegan could this be refactor to use the `resource` actions. In this case show (or index not sure what it does) ?
perhaps using singular resource if there is nothing that can be used as an ID https://guides.rubyonrails.org/routing.html#singular-resources

Reply:
I had it as a resource action initially, but in this case it's not really a proper rest resource. the queries are

/ami?chartType='blah'&year='blah' it didn't really seem like an index to me, and it doesn't have an id so doesn't make sense as a show either.

Reply:
@fddayan any more feedback here? does it still seem like it should be an index to you?